test/export/mcap_test.py

```diff
@@ -0,0 +1,221 @@
+    # Create road network model
+    env = os.getenv("MALIPUT_MALIDRIVE_RESOURCE_ROOT").split(":")
+    env = env[0]
+    rn_configuration = {
+        "opendrive_file": env + "/resources/odr/TShapeRoad.xodr" if env is not None else "",
+        "road_geometry_id": "TShapeRoadRoadNetwork"
+    }
+    maliput_road_network = create_road_network(
+        "maliput_malidrive", rn_configuration)
+    return maliput_road_network
+
+
+def GetSimulationMock(simulation_states: List[SimulationState], road_network):
+    """
+    Returns a mock simulation with the given simulation states and road network.
+
+    Args:
+        simulation_states: The simulation states to be returned by the mock simulation.
+        road_network: The road network to be returned by the mock simulation.
+    """
+    mock_simulation = MagicMock(spec=Simulation)
+    mock_simulation.get_sim_states.return_value = simulation_states
+    mock_simulation.get_road_network.return_value = road_network
+    return mock_simulation
+
+
+def GetSimulationState(sim_time: float):
+    """
+    Returns a simulation state at a given simulation time.
+    The ecm state contains a road network entity and an agent entity.
+    The velocity of the agent is set to 1.0 in the x direction.
+    """
+    linear_vel_x = 1.
+    return SimulationState(
+        sim_time=sim_time,
+        ecm_state={'entities':
+                   [
+                       {
+                           'id': 'entity_1',
+                           'components':
+                           {
+                               'Name': ['maliput'],
+                               'Type': ['road_network'],
+                               'RoadNetwork': [None]
+                           }
+                       },
+                       {
+                           'id': 'entity_2',
+                           'components':
+                           {
+                               'Name': ['agent_1'],
+                               'Type': ['agent'],
+                               'Pose': [[linear_vel_x * sim_time, 0.0, 0.0, 0.0, 0.0, 0.0, 1]],
+                               'Velocity': [[1.0, 0, 0, 0, 0, 0.0]],
+                               'Behavior': [{'signal_1': 'value_1'}]
+                           }
+                       },
+                   ]
+                   })
+
+
+class TestMCAP(unittest.TestCase):
+    _SIM_STATE_EMPTY = SimulationState(sim_time=0.0, ecm_state={})
+
+    def test_init_method(self):
+        simulation = GetSimulationMock([self._SIM_STATE_EMPTY], None)
+        dut = MCAP(simulation)
+        self.assertEqual(dut._simulation, simulation)
+        self.assertEqual(dut._output_file_path, MCAP._DEFAULT_SIMULATION_OUTPUT_FILE_PATH)
+        dut = MCAP(simulation, "random_path.mcap")
+        self.assertEqual(dut._output_file_path, "random_path.mcap")
+
+    def test_add_road_network_entity_to_scene_method(self):
```

Comment:
Do you see value in adding the `_method`?

Reply:
Removing.